src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -122,15 +151,17 @@ public Uri SourceUri
             string source,
             string apiKey,
             bool noServiceEndpoint,
+            SymbolPackageUpdateResourceV3 symbolPackageUpdateResource,
             TimeSpan requestTimeout,
             ILogger log,
             CancellationToken token)
         {
+            var isSymbolEndpointSnupkgCapable = symbolPackageUpdateResource != null;
             // Get the symbol package for this package
-            var symbolPackagePath = GetSymbolsPath(packagePath);
+            var symbolPackagePath = GetSymbolsPath(packagePath, isSymbolEndpointSnupkgCapable);
 
             // Push the symbols package if it exists
-            if (File.Exists(symbolPackagePath))
+            if (File.Exists(symbolPackagePath) || symbolPackagePath.IndexOf('*') != -1)
```

Comment:
what does `*` means here?

Reply:
you can specify *.nupkg in push command.